articles/aks/azure-disk-csi.md

```diff
@@ -308,6 +308,10 @@ Expand the PVC by increasing the `spec.resources.requests.storage` field running
 kubectl patch pvc pvc-azuredisk --type merge --patch '{"spec": {"resources": {"requests": {"storage": "15Gi"}}}}'
 ```
 
+> [!NOTE]
+> Shrinking of persistent volumes is currently not supported. Trying to patch an existing PVC with a smaller size then the current one will lead to the following error message:
```

Comment:
To improve grammar and formatting, I'm suggesting the following changes:

Shrinking persistent volumes is currently not supported. Trying to patch an existing PVC with a smaller size than the current one leads to the following error message:
"`The persistent volume claim "pvc-azuredisk" is invalid: spec.resources.storage: Forbidden: field can not be less than previous value.`"

Remove double-quotes before/after single tick mark to ensure the error message is wrapped in a code-block style format.

Reply:
Adjusted accordingly!

Reply:
Adjusted accordingly!